README.md

```diff
@@ -26,20 +32,21 @@
 <p align="center"><i>This library is still under development. Expect breaking changes!</i></p>
 <br/>
 
-`dioxus-std` is a Dioxus standard library that provides abstractions for your Dioxus app. Abstractions included are notifications, clipboard, and more to come.
+`dioxus-std` is a Dioxus standard library that provides abstractions for your Dioxus app. Abstractions included are notifications, clipboard, geolocation and storage with more to come!
```

Comment:
SDK refers to "software development kit" whereas the README intro says "standard library". Is it not confusing in a way?

Reply:
This PR will be for the ``dioxus-std`` crate. Just the notice of rename and 0.5 updates to the README. There will be another PR that renames everything to SDK and ``dioxus-sdk`` for the new crate.